app/code/Magento/Backend/App/AbstractAction.php

```diff
@@ -83,7 +83,17 @@
      * @var \Magento\Framework\Data\Form\FormKey\Validator
      */
     protected $_formKeyValidator;
-
+    
+    /**
+     * @var \Magento\Framework\Serialize\Serializer\Json
+     */
+    protected $_jsonSerializer;
```

Comment:
Please, make properties private

Reply:
And avoid leading underscores.

Reply:
@sidolov Why and when should I use private properties rather than protected ones? Since this is an abstract class children classes could make use of it, couldn't they?

Reply:
> replace the underscore of all the properties that already exist in the core modules?

It would be backward-incompatible. For now we just respect PSR-2 for new properties.

> Since this is an abstract class children classes could make use of it, couldn't they?

The need for inheritance must be eliminated actually.